esmvalcore/preprocessor/_regrid.py

```diff
@@ -151,25 +192,95 @@ def _stock_cube(spec, lat_offset=True, lon_offset=True):
         londata = np.linspace(_LON_MIN, _LON_MAX - dlon,
                               int(_LON_RANGE / dlon))
 
-    lats = iris.coords.DimCoord(
-        latdata,
-        standard_name='latitude',
-        units='degrees_north',
-        var_name='lat')
-    lats.guess_bounds()
+    cube = _stock_cube(latdata=latdata, londata=londata)
 
-    lons = iris.coords.DimCoord(
-        londata,
-        standard_name='longitude',
-        units='degrees_east',
-        var_name='lon')
-    lons.guess_bounds()
+    return cube
 
-    # Construct the resultant stock cube, with dummy data.
-    shape = (latdata.size, londata.size)
-    dummy = np.empty(shape, dtype=np.dtype('int8'))
-    coords_spec = [(lats, 0), (lons, 1)]
-    cube = iris.cube.Cube(dummy, dim_coords_and_dims=coords_spec)
+
+def _spec_to_latlonvals(*, lat_start: float, lat_end: float, lat_step: float,
+                        lon_start: float, lon_end: float,
+                        lon_step: float) -> tuple:
+    """Define lat/lon values from spec.
+
+    Create a regional cube starting defined by the target specification.
+
+    The latitude must be between -90 and +90. The longitude is not bounded, but
+    wraps around the full great circle.
+
+    Parameters
+    ----------
+    lat_start : float
+        Latitude value of the first grid cell center. The grid includes this
+        value.
+    lat_end : float
+        Maximum latitude value of the last grid cell. The grid includes this
+        value only if it falls on a grid point. Otherwise, it cuts off at the
+        first previous value.
+    lat_step : float
+        Latitude spacing between grid points, i.e. the distance between two
+        adjecent values.
```

Comment:
is this the distance between the centers of two neighboring cells? if so, probably better to say it that way